datasets/metrec/README.md

```diff
@@ -0,0 +1,151 @@
+---
+annotations_creators:
+- no-annotation
+language_creators:
+- found
+languages:
+- ar
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 55K
+source_datasets:
+- original
+task_categories:
+- poetry_classification
+task_ids:
```

Comment:
```suggestion
task_categories:
- text-classification
task_ids:
- text-classification-other-poetry-classification
```

please use the tagger https://github.com/huggingface/datasets-tagging to see which categories are available